doc/topics/releases/fluorine.rst

```diff
@@ -624,3 +624,14 @@ New style for including from a parent directory:
 .. code-block:: jinja
 
     {% from '../foo' import bar %}
+
+salt-api
+============================
+
+salt-api Windows support
+--------------------------------
+
+Previously, salt-api was was not supported on the Microsoft Windows platforms. Now it is!
+salt-api is providing a RESTful interface to a running Salt system. It allows
+for viewing minions, runners, and jobs as well as running execution modules
+and runners of a running Salt system through a REST API that returns JSON.
```

Comment:
Adding a link to the Salt-API documentation might be a good idea here. :)

Reply:
Regarding Salt-API documentation, do you mean:
https://salt.readthedocs.io/projects/salt-api
or
https://docs.saltstack.com/topics/netapi/index.html
or something else?

Reply:
Ah, sorry I was unclear. The second one: https://docs.saltstack.com/en/latest/topics/netapi/index.html